drupal/web/modules/custom/wunder_next/wunder_next.install

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the wunder_next module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function wunder_next_install() {
+
+  // Currently these configs cannot be set via recipes, so we
```

Comment:
This is a bit funky, but no other way at the moment...

Reply:
Drupal slack to the rescue! I have replaced this with the correct way of updating existing configuration.